acceptance/openstack/css/v1/clusters_test.go

```diff
@@ -13,6 +14,14 @@ import (
 
 const timeout = 1200
 
+func getEnvVar(key string) string {
```

Comment:
dont need to introduce this, we already have clients.EnvOS.GetEnv(name)

Reply:
Function removed.